pytorch_lightning/trainer/training_loop.py

```diff
@@ -301,6 +303,12 @@ def has_arg(self, *args):
     def train(self):
         rank_zero_warn('Displayed epoch numbers in the progress bar start from "1" until v0.6.x,'
```

Comment:
I think that we can remove this warning

Reply:
I will do it in a separate PR, since It will be easier to track changes that way :)